src/Tickets/Emails/Hooks.php

```diff
@@ -64,6 +66,21 @@ protected function add_filters() {
 
 		// Hook the `Tickets Emails` preview for the AJAX requests.
 		add_filter( 'tribe_tickets_admin_manager_request', [ $this, 'filter_add_preview_modal_content' ], 15, 2 );
+
+		add_filter( 'wp_redirect', [ $this, 'filter_redirect_url' ] );
+	}
+
+	/**
+	 * Filters the redirect URL to determine whether or not section key needs to be added.
+	 *
+	 * @since TBD
+	 *
+	 * @param string $url Redirect URL.
+	 *
+	 * @return string
+	 */
+	public function filter_redirect_url( $url ) {
+		return $this->container->make( Emails_Tab::class )->filter_redirect_url( $url );
```

Comment:
Unless something has changed since we added the Tickets Commerce tab, the section doesn't automatically get appended when saving the settings. In my testing, this was the case and this filter is needed.